controllers/ccruntime_controller.go

```diff
@@ -331,7 +331,8 @@ func (r *CcRuntimeReconciler) updateUninstallationStatus(finishedNodes int) (ctr
 }
 
 func handlePostUninstall(r *CcRuntimeReconciler) (ctrl.Result, error) {
-	err, nodes := r.getNodesWithLabels(map[string]string{"cc-postuninstall/done": "true"})
+	postUninstallDoneLabel := map[string]string{PostUninstallDoneLabel[0]: PostUninstallDoneLabel[1]}
```

Comment:
Little rant, not your fault: it would be better to be used string variables like `PostUninstallDoneLabel` and `PostUninstallDoneLabelValue` .. as currently  `PostUninstallDoneLabel` is a fixed slice of two read-only values....

Reply:
could you please elaborate a bit? I'm not sure I understand that properly...